examples/DrawXNative/DrawXShared/DrawPath.cs

```diff
@@ -0,0 +1,38 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Realms;
+using SkiaSharp;
+
+namespace DrawXShared
+{
+    [SuppressMessage(
+        "Microsoft.StyleCop.CSharp.NamingRules",
+        "SA1300:ElementMustBeginWithUpperCaseLetter", 
+        Justification = "Conform to same schema as other Realm samples")]
+    public class DrawPath : RealmObject
+    {
+        public string drawerID { get; set; }
```

Comment:
I thought `MapTo` was still undecided? I think it shows a poor example if every single property has it.